lib/Bio/EnsEMBL/DataCheck/index.json

```diff
@@ -1113,6 +1113,15 @@
       "name" : "DensitySNPs",
       "package_name" : "Bio::EnsEMBL::DataCheck::Checks::DensitySNPs"
    },
+   "DisplayNameFormat" : {
+      "datacheck_type" : "critical",
+      "description" : "For Rapid Release, the display name must be a specific format",
+      "groups" : [
+         "rapid_release"
```

Comment:
How would rapid_release set to be run only for RR handover?

Reply:
The datacheck execution part of the handover service now has a concept of the target server, with the 'staging_uri' parameter (https://github.com/Ensembl/ensembl-prodinf-handover/blob/main/src/ensembl/production/handover/celery_app/utils.py#L224). So could use that to add the 'rapid_release' group to the set of datachecks that are executed.

Reply:
Awesome!